views/errordisplay.ejs

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>SIGN UP</title>
+  <link rel="stylesheet" type="text/css" href="/css/style.css" />
+</head>
+
+<body>
+  <form action="/signup" method="POST" class="forms">
+  <h4>SIGN UP</h4>
+      <label for="username"><b>Username</b></label>
+      <input type="text" placeholder="Enter your Name" id="username" name="username">
+      <br>
+      <label for="password"><b>Password</b></label>
+      <input type="password" placeholder="Enter your password" id="password" name="password">
+      <br>
+      <label for="email"><b>Email ID</b></label>
+      <input type="email" placeholder="Enter your Email ID" id="email" name="email">
+      <br>
+      <button type="submit" class="submit">SUBMIT</button>
+  </form>
+
+  <h5>PLEASE ENTER VALID CREDENTIALS</h5>
+  <% if(typeof alert != 'undefined') {%>
```

Comment:
Is there a type here ?  on line 33 there is another variable used 'alerts' (plural).

Reply:
If the credentials given by user is invalid,An array of errors will be stored in the alert. if so the variable alert cannot have an undefined type.That's why typeof alert is used.

Reply:
I meant is  Alert is the array of errors and alerts is the string passed. so type of alerts cannot be undefined.

Reply:
use different variable names then.  It will avoid the confusion

Reply:
Okay manoj.